src/components/calcite-action-pad/calcite-action-pad.tsx

```diff
@@ -186,6 +188,12 @@ export class CalciteActionPad {
       />
     ) : null;
 
+    if (rtl) {
+      if (expandToggleEl?.shadowRoot) {
+        expandToggleEl.shadowRoot.querySelector("button").classList.add(CSS_UTILITY.rtl);
```

Comment:
I feel like we've run into issues with querying shadow root before, am I making that up? Maybe @eriklharper knows...

Reply:
The only place I can remember doing this kind of thing is in `calcite-slider` but I'm realizing now that grabbing refs on the elements I'm querying for is a better approach from a performance standpoint.  I don't recall any 'issues' with doing this in the past, though within component code.

Reply:
Yeah, we should avoid querying at all if it's possible to achieve using `ref` and storing the node. Querying shadow DOM should be avoided as much as possible as well because in general, internals of another component shouldn't be accessed from outside.

Reply:
Thanks for your help! @driskull just recommended a ridiculously easy fix by setting dir on CalciteExpandToggle's calcite-action. I didn't realize it was that easy 🤦‍♀️